DOCPLAN-100: Vale updates for /virt/support/virt-troubleshooting.adoc (#108585)
openshift/openshift-docs: modules/virt-enable-guest-log-default-web.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enable-guest-log-default-web_{context}"]
= Enabling default access to guest system logs with the web console

[role="_abstract"]

> To more easily diagnose and troubleshoot issues, you can enable default access to virtual machine (VM) guest system logs using the web console.

Reviewer: ```suggestion
To troubleshoot issues more easily, you can enable default access to virtual machine (VM) guest system logs by using the web console.
```

Same suggestion as the previous module, but also, why did you remove the `by` in `by using`? Am I missing new guidance?

Author: Done. No guidance change I am aware of here. I was using Ingrid's NotebookLM short description generator as the basis of the changes.
